dockerfile/README.md

```diff
@@ -0,0 +1,42 @@
+# Dockerfile for WSO2 Data Analytics Server #
+
+The Dockerfile defines the resources and instructions to build the Docker image for WSO2 Data Analytics Server 3.1.0.
+
+## Quickstart guide
+
+ Steps to build the WSO2 Data Analytics Server 3.1.0 Docker image and run in your local machine are as follows:
+ 
+ The local copy of the `Dockerfile` directory will be referred as, `DOCKERFILE_HOME`.
+ 
+ * Create a directory named `files` inside `DOCKERFILE_HOME`
+     - This will result in a directory structure `<DOCKERFILE_HOME>/files`.
+ 
+ * Add the JDK and WSO2 API Manager distributions
+     - Download JDK 1.8 (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and copy it to `<DOCKERFILE_HOME>/files`.
+     - Download the WSO2 Data Analytics Server 3.1.0 distribution (https://wso2.com/analytics) and copy it to `<DOCKERFILE_HOME>/files`.
+ 
+ * Build the Docker image
+     - Navigate to `<DOCKERFILE_HOME>` directory.
+     - Execute the `docker build` command as shown below;
+         + `docker build -t wso2das:3.1.0 files/`
+ 
+ * Docker run
+     - Run the WSO2 Data Analytics Server 3.1.0 Docker container as follows;
+         + `docker run -it wso2das:3.1.0`
```

Comment:
We might need to mention the port mappings in the docker run command. Otherwise, on platforms such as OSX and Windows where Docker is not run on the same host machine, the container ports will not be accessible.